source/guide_goaccess.rst

```diff
@@ -0,0 +1,199 @@
+
+.. author:: FM <git.fm@mmw9.de>
+
+########
+GoAccess
+########
+
+GoAccess_ is an open source realtime web log analyzer and viewer. It use the standard web server log file as source to build a readable graphical HTML output as web page (dashboard) or an overview in the shell. 
+
+This documentation describe the way to let create a static web page as overview, which will be updated every hour with a cron job. More details are on the `man page`_ and on GitHub_ available.
+
+License
+=======
+
+GoAccess is released under the `MIT License`_
+
+Installation
+============
+
+Step 1 - Download and Extract the Source Code
+---------------------------------------------
+
+Create a working directory.
+
+::
+
+ [isabell@stardust ~]$ mkdir ~/filebase/
+ [isabell@stardust ~]$ cd ~/filebase/
+ [isabell@stardust ~]$
+
+Download the actual source code from GitHub. 
+
+::
+
+ [isabell@stardust ~]$ git clone https://github.com/allinurl/goaccess.git
+ [isabell@stardust ~]$
+
+Step 2 - Source Code Configuration, Compiling and Installation
+--------------------------------------------------------------
+
+Prepare the compile environment.
+
+::
+
+ [isabell@stardust ~]$ cd ~/filebase/goaccess/
+ [isabell@stardust ~]$ autoreconf -fiv
+ [isabell@stardust ~]$
+
+Configure and compile the source code with the commands ``configure``, ``make`` and finally install it with ``make install``.
+
+.. note:: Please use single steps instead of combining all three in one process to see and identify possible errors.
+
+Before we start, we have to consider some aspects, especially a shared hosting environment like Uberspace:
+
+ * ``--prefix=/home/isabell``: Installation target for your personal Uberspace.
+ * ``--enable-utf8``: Compile with wide character support.
+ * ``--enable-geoip=legacy``: Compile with GeoLocation support, legacy will utilize the original GeoIP databases.
+
+Other options can be found in the GoAccess `installation documentation`_.
+
+.. warning:: Replace ``isabell`` with your Uberspace name!
+
+::
+
+ [isabell@stardust ~]$ cd ~/filebase/goaccess/
+ [isabell@stardust ~]$ ./configure --enable-utf8 --enable-geoip=legacy --prefix=/home/isabell
+ [isabell@stardust ~]$ make
+ [isabell@stardust ~]$ make install
+ [isabell@stardust ~]$
+
+Configuration
+=============
+
+After the installation of GoAccess, it is necessary to enable the web server logs and to configure GoAccess with the right log format.
+
+Step 1 - Enable the Web Server Log
+----------------------------------
+
+Please follow the instructions by the Uberspace manual_ to enable the web server logs.
+
+.. note:: Please consider, after the web server log enabling, it needs some time to have some entries in your new log files. Depends on the web traffic.
+
+Step 2 - GoAccess Configuration
+-------------------------------
+
+Edit the configuration file ``~/etc/goaccess/goaccess.conf`` and reach out for the following parameters to uncomment these.
+
+.. code-block:: bash
+ :emphasize-lines: 2,4,6,8
```

Comment:
These [do not seem to match up](https://deploy-preview-539--uberlab.netlify.com/guide_goaccess.html#step-2-goaccess-configuration) with the changed config lines anymore.